### PR TITLE
LPS-194850 Allow Object Validation Rules to be updated

### PR DESCRIPTION
**Pull request by mateussgn**
Related issue: https://liferay.atlassian.net/browse/LPS-194850

### CODE REVIEW

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleSettingLocalServiceImpl.java [hunk 1]
```diff
@@ -53,6 +60,155 @@ public ObjectValidationRuleSetting addObjectValidationRuleSetting(
...
+				objectValidationRuleSettings) {
+
+			if (objectValidationRuleSetting.compareName(
+					ObjectValidationRuleSettingConstants.
+						NAME_KEY_OBJECT_FIELD_ID)) {
+
+				keyObjectValidationRuleSettings.add(
+					objectValidationRuleSetting);
+			}
+			else {
```
Comment (pedro-oliveira446) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleSettingLocalServiceImpl.java:205]: Here can we use the `continue` statement instead `else` to keep the same pattern ?

**Review by pedro-oliveira446 — APPROVED** [review]
Great job @mateussgn, 
I just have a quick question.

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleSettingLocalServiceImpl.java (hunk 1 above)
Comment (mateussgn) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleSettingLocalServiceImpl.java:205]: Sure!
Thanks for the tip!

**Comment by gabrielwas** [comment]
Blocked by https://github.com/liferay-objects/liferay-portal/pull/2596

**Comment by mateussgn** [comment]
I'll close this pull request due to code improvements that I observed.

**Comment by mateussgn** [comment]
ci:close

**Pull request closed without merging**